Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -359,28 +411,49 @@ public override void PutTransaction<T>(Transaction<T> tx)
             }
 
             byte[] key = TxKey(tx.Id);
-
-            if (!(_txDb.Get(key) is null))
+            if (!(_txIndexDb.Get(key) is null))
             {
                 return;
             }
 
-            _txDb.Put(key, tx.Serialize(true));
+            var timestamp = tx.Timestamp.ToUnixTimeSeconds();
+            string txDbName = $"epoch{(int)timestamp / _txAtEachEpoch}";
+            RocksDBStoreEpoch epoch = new RocksDBStoreEpoch("Transaction", txDbName);
```

Comment:
IMO, `RocksDBStoreEpoch` can be replaced `blockDbName` or `txDbName`. if then, we don't need to serialize/deserialize `RocksDBStoreEpoch`, but just need to load/store its value from rocksdb.

Reply:
you're right, it is useless structuring.